test/parser.test.ts

```diff
@@ -129,7 +129,7 @@ describe("parser tests", () => {
 
     it("should compute the correct hash id", () => {
       const combat = combats[0];
-      expect(combat.id).toEqual("f3750ed46db5cabc1d25882e6fa2c67b");
+      expect(combat.id).toEqual("65801fbacf7700cc1fa3744ecaffd4a2");
```

Comment:
Can you include the original test + original hash on this? Changing both input and output scares me a little as I want to be sure the hashing wasn't affected by this PR.

Reply:
that's a good point! done in the latest commit